unittest/frames.cpp

```diff
@@ -7,13 +7,16 @@
 #include "pinocchio/algorithm/jacobian.hpp"
 #include "pinocchio/algorithm/frames.hpp"
 #include "pinocchio/algorithm/rnea.hpp"
+#include "pinocchio/algorithm/crba.hpp"
```

Comment:
Pinocchio might not have been compiled with URDFDOM support, could you add `#ifndef PINOCCHIO_WITH_URDFDOM_SUPPORT` to make the code compile if urdfdom is missing?

Reply:
In the case it has not been compiled with it, should I find another way to test or just skip this test ?

Reply:
Using it, you won't need anymore to rely on urdfdom and just build dummy models using the current helpers in Pinocchio.

Reply:
https://github.com/stack-of-tasks/pinocchio/pull/1797